app/views/shared/_is_applying_for_coverage.html.erb

```diff
@@ -19,13 +19,13 @@
   <div class="col-lg-2 col-md-2 col-sm-2 col-xs-2 form-group form-group-lg no-pd">
     <div tabindex="0" onkeydown="handleRadioKeyDown(event, 'is_applying_coverage_true')" class="radio">
       <%= f.radio_button :is_applying_coverage, true, class: "required floatlabel", id: 'is_applying_coverage_true', checked: first_checked%>
-      <label for="is_applying_coverage_true"><span class="yes_no_pair"><%= l10n("yes") %></span></label>
+      <label for="is_applying_coverage_true"><span class="yes_no_pair">Yes</span></label>
     </div>
   </div>
   <div class="col-lg-2 col-md-2 col-sm-2 col-xs-6 form-group form-group-lg no-pd">
     <div tabindex="0" onkeydown="handleRadioKeyDown(event, 'is_applying_coverage_false')" class="radio">
       <%= f.radio_button :is_applying_coverage, false, class: "required floatlabel", id: 'is_applying_coverage_false', checked: second_checked %>
-      <label for="is_applying_coverage_false"><span class="yes_no_pair"><%= l10n("no") %></span></label>
+      <label for="is_applying_coverage_false"><span class="yes_no_pair">No</span></label>
```

Comment:
any reason why we removed the l10n here?

Reply:
same as above